payments/invoice/other-features.md

```diff
@@ -233,53 +515,84 @@ Content-Type: application/json
+
+The `authorization` resource contains information about an authorization
+transaction made towards a payment, as previously described.
 
-When performing unsuccessful operations, the eCommerce API will respond with
-a problem message. We generally use the problem message type and status code
-to identify the nature of the problem. The problem name and description will
-often help narrow down the specifics of the problem.
+{% include transactions-reference.md %}
 
-#### Error types from Swedbank Pay Invoice and third parties
+{% include callback-reference.md %}
 
-All invoice error types will have the following URI in front of type:
-`https://api.payex.com/psp/errordetail/invoice/<errorType>`
+{% include payment-link.md hide-3d-secure=true %}
 
-{:.table .table-striped}
-| **Type**        | **Status** |
-| :-------------- | :--------- |
-| *externalerror* | 500        | No error code                 |
-| *inputerror*    | 400        | 10 - ValidationWarning        |
-| *inputerror*    | 400        | 30 - ValidationError          |
-| *inputerror*    | 400        | 3010 - ClientRequestInvalid   |
-| *externalerror* | 502        | 40 - Error                    |
-| *externalerror* | 502        | 60 - SystemError              |
-| *externalerror* | 502        | 50 - SystemConfigurationError |
-| *externalerror* | 502        | 9999 - ServerOtherServer      |
-| *forbidden*     | 403        | Any other error code          |
+## PayeeReference
 
-{% include settlement-reconciliation.md %}
+{% include payeeinfo.md %}
 
-{% include payment-link.md %}
+## Prices
 
-{% include callback-reference.md %}
+{% include prices.md %}
 
-## Payment Orders
+{% include settlement-reconciliation.md %}
 
-{% include payment-order-get.md %}
+## Problems
 
-{% include transactions-reference.md %}
+When performing unsuccessful operations, the eCommerce API will respond with a
+problem message. We generally use the problem message `type` and `status` code
+to identify the nature of the problem. The problem `name` and `description` will
+often help narrow down the specifics of the problem.
 
-{% include operations-reference.md %}
+### Error types from Swedbank Pay Invoice and third parties
 
-### Payee Info
+All invoice error types will have the following URI in front of type:
+`https://api.payex.com/psp/errordetail/invoice/<errorType>`
 
-{% include payeeinfo.md %}
+{:.table .table-striped}
+| Type            | Status |                               |
+| :-------------- | :----- | :---------------------------- |
+| *externalerror* | 500    | No error code                 |
+| *inputerror*    | 400    | 10 - ValidationWarning        |
+| *inputerror*    | 400    | 30 - ValidationError          |
+| *inputerror*    | 400    | 3010 - ClientRequestInvalid   |
```

Comment:
What is `3010` and why is it documented here? Is it something our customers have any knowledge of?

Reply:
I do not know.

Reply:
@arebra, do you know? If `3010` and similar error codes is internal to PxR, we should probably not document those on developer portal, no?

Reply:
While it might be internal to PxR, if it is returned to the client should it not be in table?

Reply:
Those are PxR error codes and the appear in the json output. I usually end up having to ask PxR what they mean if they're not the most common ones. Wouldn't documenting them lead to less questions to support?

Reply:
I don't think `3010` and the other numbers are something we actually have in our responses. I think they are from internal mapping that are invisible to external consumers of our API. That's why I ask.

Reply:
If I read the table correctly, the user will get something like this:
```json
{
  "type": "https://api.payex.com/psp/errordetail/invoice/inputerror",
  "status": 400,
  "description": "3010 - ClientRequestInvalid"
}
```

Reply:
There shouldn't be a `description` property in `problem+json`, only `title` and `detail`. And if that's the `detail` we respond with, it's terrible as it should be human readable and explain what can be done to recover from the problem, not contain programmatic error identifiers that invite integrators to do string searches in order to interpret our problem messages. Can you investigate how our `problem+json` responses look like for invoice payments?